File: 프로그래머스/unrated/181930. 주사위 게임 2/주사위 게임 2.py
def solution(a, b, c):
    
    if a != b and b != c and a != c:
        answer = a + b + c
    elif (a == b and b != c) or (a != b and b == c) or (a == c and b != c):
        answer = (a + b + c) * (a**2 + b**2 + c**2)
    elif a == b == c:
        answer = (a + b + c) * (a**2 + b**2 + c**2) * (a**3 + b**3 + c**3)
        
    return answer

File: 프로그래머스/0/181930. 주사위 게임 2/주사위 게임 2.java
class Solution {
    public int solution(int a, int b, int c) {
        int sum = a + b + c;
        if (a != b && b != c && c != a) {
            return sum;
        } else if ((a == b && b != c) || (a != b && b == c) || (a == c && b != c)) {
            return sum * (a * a + b * b + c * c);
        } else {
            return sum * (a * a + b * b + c * c) * (a * a * a + b * b * b + c * c * c);
        }
    }
}
